selfdrive/car/volkswagen/interface.py

```diff
@@ -25,7 +25,7 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None):
     # VW port is a community feature, since we don't own one to test
     ret.communityFeature = True
 
-    if candidate in [CAR.GOLF, CAR.AUDI_A3]:
+    if candidate in MQB_CARS:
```

Comment:
remove this if, same comment as above

Reply:
Same comment as above: I request that we keep it in.